pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_importing-content.adoc

```diff
@@ -1,15 +1,15 @@
 [id="importing-(ContentTerm}_{context}"]
 
-= Importing {ContentTerm}
+= Importing content to Pantheon
 
 [role="_abstract"]
 You can import modules and assemblies into {ProductShortName} from either GitHub or GitLab.
 
 .Prerequisites
 
 //* You are assigned a publisher or author role in {ProductShortName}.
-* You are logged in.
-* You obtained the Git repository URL for the {ContentTerm} you want to import (For example, https://github.com/openshift/openshift-docs).
+* You are logged in to {ProductShortName}.
+* You have a URL to the Git repository with your {ContentTerm}. For example, +https://github.com/openshift/openshift-docs+.
```

Comment:
Are you sure it's + here? Did you try it out with Asciidoctor? I think it's a backslash at the beginning of the URL to prevent it from becoming a link.

Reply:
Both approaches work. But I changed it to a backslash as the asciidoc manual recommends it.